frontend/app/routes/$lang+/_public+/apply+/$id+/adult-child/applicant-information.tsx

```diff
@@ -54,7 +54,9 @@ export async function loader({ context: { session }, params, request }: LoaderFu
   const csrfToken = String(session.get('csrfToken'));
   const meta = { title: t('gcweb:meta.title.template', { title: t('apply-adult-child:applicant-information.page-title') }) };
 
-  return json({ id: state.id, maritalStatuses, csrfToken, meta, defaultState: state.applicantInformation, editMode: state.editMode });
+  const ageCategory = getAgeCategoryFromDateString(state.dateOfBirth ?? '');
```

Comment:
Use `invariant` to ensure `dateOfBirth` is not undefined.

```suggestion
  invariant(state.dateOfBirth, 'Expected state.dateOfBirth to be defined');
  const ageCategory = getAgeCategoryFromDateString(state.dateOfBirth);
```